tests/components/ness_alarm/test_init.py

```diff
@@ -261,7 +260,12 @@ def mock_nessclient():
     _mock_factory = MagicMock()
     _mock_factory.return_value = _mock_instance
 
-    with MockDependency("nessclient"), patch(
-        "nessclient.Client", new=_mock_factory, create=True
-    ), patch("nessclient.ArmingState", new=MockArmingState):
+    with patch(
+        "homeassistant.components.ness_alarm.Client", new=_mock_factory, create=True
+    ), patch(
+        "homeassistant.components.ness_alarm.ArmingState", new=MockArmingState
```

Comment:
`homeassistant.components.ness_alarm.ArmingState` and `homeassistant.components.ness_alarm.alarm_control_panel.ArmingState` are the same, is it necessary to patch both? Does it work with just `homeassistant.components.ness_alarm.ArmingState`

Reply:
Unfortunately not. Patched only one of them which resulted in the tests failing (https://github.com/home-assistant/home-assistant/pull/29518/commits/3aec0c91e73125d534df5e7c46efd5aacfca2465). Therefore I would say both are necessary.

Reply:
@nickw444 so what do you say? :)